libraries/core_libs/consensus/src/transaction/transaction_manager.cpp

```diff
@@ -354,6 +362,10 @@ std::pair<SharedTransactions, std::vector<uint64_t>> TransactionManager::packTrx
     total_weight += weight;
     if (total_weight > weight_limit) {
       trxs.resize(i);
+      total_weight -= weight;
+      if(weight_limit - total_weight >= 21000) {
```

Comment:
kMinTxGas should be used instead of 21000

Reply:
Done